tox.ini

```diff
@@ -89,8 +89,8 @@ exclude_lines = if __name__ == ["']__main__["']:
 
 [coverage:paths]
 source = tox
-         {toxworkdir}/*/lib/python*/site-packages/tox
-         {toxworkdir}/*/Lib/site-packages/tox
+         {toxworkdir}/*/lib/python*/site-packages/tox"
```

Comment:
at first glance the ones in the front seem to be missing

Reply:
doh.... sorry... those seem to have been left in by mistake - nothing needs to be quoted there - fixing it now